### DISCUSSION TITLE
Android SDK - Conference Activity - App Crash

### DISCUSSION THREAD

**Discussion by AndroidApprikart** [ant-media/Ant-Media-Server#5468] [Q&A]
Our team is trying to use the Video Conference section in the sample app attached with the latest version of Android SDK. 
Scenario:
- The app is running on 2 phones say Phone A and Phone B
- in Phone A ->  the user has started publishing with the Conference section, the stream is being published successfully.
- Now,  when the Phone B user tries to join the vc room , the app crashes.

Please help  the log looks like the following
![Screenshot from 2023-08-17 12-56-51](https://github.com/ant-media/Ant-Media-Server/assets/109220606/13c70106-f139-4f54-84c4-a34a91aaa314)

**Comment by USAMAWIZARD** [comment]
@AndroidApprikart let me try to reproduce this error and get back to you soon

**Comment by USAMAWIZARD** [comment]
@AndroidApprikart yes you are right ,  there I seems to be some problem with it let me create an issue from this discussion

**Comment by lastpeony** [comment]
Crash is happening due to  AbstractSDKSampleActivity.java onNewVideoTrack methods Toast call. Call it inside runOnUiThread and app wont crash.
For now please use Multitrack conference sample. Conference is deprecated. Multitrack conference sample is working fine.
